src/apps/basic/basic_apps.lua

```diff
@@ -86,6 +87,90 @@ function Sink:push ()
    end
 end
 
+--- ### `Match` app: Compare packets recieved on rx and comparator
+Match = {}
+
```

Comment:
I think `Match` is a bit too fancy for `apps/basic/basic_apps.lua` and would move it two its own module `apps/match/match.lua`.